docs/user/01-20-command-palette.md

```diff
@@ -0,0 +1,26 @@
+# Command Palette
+
+With the Kyma dashboard command palette feature, you can easily navigate through all your dashboard resources, such as workloads, deployments, etc.
+You can perform actions such as listing or accessing your namespaces, or preferences, or uploading YAML files.
+
+## Features
+
+You can use command palette to perform the following actions:
+
+- Access command palette, by pressing `cmd+k` on your keyboard or choosing **Quick Navigation**.
+
+  ![Quick navigation](assets/quick-navi.png)
+
+- Enter `?` to see all navigation options and the resources' aliases. For example, if you want to see your namespaces, it's enough to type `ns`.
+
+  ![Namespaces in navigation](assets/navigation-ns.png)
+
+- See the history of your actions by using the up or down arrow. Pressing the right arrow autocompletes your search.
+
+- See the list of all resources available in the given namespace by entering the resource name and pressing the tab key.
```

Comment:
Maybe a good option will be to add also, that you can list your resources after clicking tab

![image](https://github.com/user-attachments/assets/27a1603d-9409-4f00-b0ab-9983d330151f)